docs/querying/operators.md

```diff
@@ -196,7 +196,12 @@ vector of fewer elements with aggregated values:
 * `quantile` (calculate φ-quantile (0 ≤ φ ≤ 1) over dimensions)
 
 These operators can either be used to aggregate over **all** label dimensions
-or preserve distinct dimensions by including a `without` or `by` clause.
+or preserve distinct dimensions by including a `without` or `by` clause. This clause
```

Comment:
While we have two singular "clause"s in the preceding sentence, I think "These clauses ..." sounds more correct in the next sentence?

Reply:
Good point, will fix.